compiler-rt/lib/scudo/standalone/vector.h

```diff
@@ -116,18 +117,19 @@ template <typename T> class VectorNoCtor {
   uptr CapacityBytes = 0;
   uptr Size = 0;
 
-  T LocalData[256 / sizeof(T)] = {};
+  T LocalData[StaticCapacity] = {};
   MemMapT ExternalBuffer;
 };
 
-template <typename T> class Vector : public VectorNoCtor<T> {
+template <typename T, size_t StaticCapacity = 10>
+class Vector : public VectorNoCtor<T, Max<size_t>(10, StaticCapacity)> {
```

Comment:
```suggestion
class Vector : public VectorNoCtor<T, StaticCapacity> {
```
It's a little bit weird to take `Max` here because the user will not be able to set the size smaller than 10 even when they think it's reasonable